### PR TITLE
Guide for Docker and WSL 2

### PR DESCRIPTION
**Pull request by Skrypt**
Do we still need to add this to a main readme file so that it gets linked? @agriffard

### CODE REVIEW

**Comment by Skrypt** [comment]
@sebastienros What needs to be changed on this one? Or should I close?

File: src/docs/guides/docker/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+cd /mnt/c/YourRepositoriesPath
+docker build -t orchard_core .
+docker run --name orchardcore -p 80:80 orchard_core
+```
+
+Now you can start your favorite browser to http://localhost and start testing your build!
+
+# References
+
+Inspiration taken from Scott Hansellman.
```
Comment (agriffard) [inline src/docs/guides/docker/README.md]: Hansellman => Hanselman

**Comment by Skrypt** [comment]
@agriffard missing your approval for merging. If you approve you can merge 😄

**Comment by deanmarcussen** [comment]
@Skrypt I think this needs to be a bit more of `how to build OrchardCore for docker` and include a link to the docker hub docs for running the default image.
And some information about how to setup `ORCHARDCORE_APP_DATA` as well.

Right now by following it, I don't think I have quite a full running orchard core docker instance

**Pull request closed without merging**